src/js/react/apps/linkedevents/components/TopicsFilter.tsx

```diff
@@ -0,0 +1,46 @@
+import { Select } from 'hds-react';
+import { useAtomValue, useAtom, useSetAtom } from 'jotai';
+
+import { topicsAtom, topicSelectionAtom, updateParamsAtom} from '../store';
+import SearchComponents from '../enum/SearchComponents';
+import ApiKeys from '../enum/ApiKeys';
+
+function TopicsFilter() {
+  const topics = useAtomValue(topicsAtom);
+  const [topicSelection, setTopicsFilter] = useAtom(topicSelectionAtom);
+  const updateParams = useSetAtom(updateParamsAtom);
+
+  const onChange = (value: any) => {
+    setTopicsFilter(value);
+    updateParams({ [ApiKeys.KEYWORDS]: value.map((topic: any) => topic.value).join(',') });
+  };
+
+  const locationHelper = Drupal.t('Filter events by topics', {}, { context: 'React search topics filter' });
```

Comment:
For some reason the string context doesn't get picked up by Drupal when the Drupal.t function is called here, so they're visible in the UI translation but without the context. Which means the translations in the `.po` files don't get imported automatically.

Seems to happen in other react components when the translated string is stored in a variable, but works correctly when called directly, like inside the Select component below: `placeholder={Drupal.t('All', {}, { context: 'React search: all available options' })}`

Reply:
Translations worked for me after running: `drush locale:check && drush locale:update; drush cr`.